### PR TITLE
FXIOS-1226 ⁃ No Issue - Trying Xcode Test Plans

### PR DESCRIPTION
**Pull request by isabelrios**
This PR is to work on the Test Plans that Xcode offers. The idea is to use Test Plans to run the Performance tests and think about removing the schemes and use Test Plans isntead if that works and simplifies the app

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXIOS-1226)

### CODE REVIEW

**Comment by isabelrios** [comment]
With these changes there are also a few more needed on Bitrise so that the current implementation works.
For the RunAllXCUITest workflow it is necessary to add the -testPlan Fennec_Enterprise_XCUITests to the xcodebuild command when the build for testing is created, in the Xcode Build for testing step.

**Comment by isabelrios** [comment]
There is the option to run tests on real devices, which is where performance test would make sense. But the latest version available at the moment on Firebase is 13.6. Even though, it may make sense to have these tests running there.

**Comment by isabelrios** [comment]
Using this PR to remove the Sync Int scheme by adding a test plan for it too.

File: XCUITests/PerformanceTestPlan.xctestplan [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      "name" : "Configuration 1",
+      "options" : {
+
+      }
+    }
+  ],
+  "defaultOptions" : {
+
+  },
+  "testTargets" : [
```
Comment (AaronMT) [inline XCUITests/PerformanceTestPlan.xctestplan:14]: How do these get updated in testplans when tests are added or removed?

**Comment by AaronMT** [comment]
Richard and I were looking at this and Xcode test-plans yesterday evening as they are new to us. How do these get executed on Bitrise?

**Review by AaronMT — APPROVED** [review]

File: XCUITests/PerformanceTestPlan.xctestplan (hunk 1 above)
Comment (isabelrios) [reply XCUITests/PerformanceTestPlan.xctestplan:14]: that happens automatically when you select the test plan from xcode and add/remove tests

**Comment by isabelrios** [comment]
> Richard and I were looking at this and Xcode test-plans yesterday evening as they are new to us. How do these get executed on Bitrise?

With this change: 

> With these changes there are also a few more needed on Bitrise so that the current implementation works.
> For the RunAllXCUITest workflow it is necessary to add the -testPlan Fennec_Enterprise_XCUITests to the xcodebuild command when the build for testing is created, in the Xcode Build for testing step.

**Pull request merged**